README.md

```diff
@@ -22,14 +22,19 @@ Every vector map tiles hoster has his own schema, so every map style is per se o
 
 ## Contributing
 
-* Open the code in Tangram Play ([Normal map](https://tangram.city/play/?scene=https://raw.githubusercontent.com/ENT8R/streetcomplete-mapstyle/jawg/streetcomplete-light-style.yaml),  [Satellite map](https://tangram.city/play/?scene=https://raw.githubusercontent.com/ENT8R/streetcomplete-mapstyle/jawg/streetcomplete-satellite-style.yaml), [Dark map](https://tangram.city/play/?scene=https://raw.githubusercontent.com/ENT8R/streetcomplete-mapstyle/jawg/streetcomplete-dark-style.yaml))
+* Open the code in an editor
+  * StreetComplete Map Style Editor
+   (recommended; [Normal map](https://streetcomplete.github.io/streetcomplete-mapstyle/?provider=jawg&style=light),  [Satellite map](https://streetcomplete.github.io/streetcomplete-mapstyle/?provider=jawg&style=satellite), [Dark map](https://tangram.city/play/?scene=https://raw.githubusercontent.com/ENT8R/streetcomplete-mapstyle/jawg/streetcomplete-dark-style.yaml))  
+   Already includes a jawg.io API key
+  * Tangram Play ([Normal map](https://tangram.city/play/?scene=https://raw.githubusercontent.com/ENT8R/streetcomplete-mapstyle/jawg/streetcomplete-light-style.yaml),  [Satellite map](https://tangram.city/play/?scene=https://raw.githubusercontent.com/ENT8R/streetcomplete-mapstyle/jawg/streetcomplete-satellite-style.yaml), [Dark map](https://tangram.city/play/?scene=https://raw.githubusercontent.com/ENT8R/streetcomplete-mapstyle/jawg/streetcomplete-dark-style.yaml))  
+    When using jawg.io, add your API key to the field `api_key` in the `global` section
 * Change or add something to the style
 * Download the file after you did your work
 * Make a pull request to this repository
 
 ## Useful links
 
+* [View map & edit map style](https://streetcomplete.github.io/streetcomplete-mapstyle)
```

Comment:
```suggestion
* [View map & edit map style](https://streetcomplete.github.io/streetcomplete-mapstyle/?provider=jawg)
```

It's better to explicitly name the provider IMO in case the default provider changes in the future...